### PR TITLE
Updates to vanilla profile section

### PR DESCRIPTION
**Pull request by itscynxx**
This has become a very common question in recent days, decided I wanted to make this clearer to make a tag for it

### CODE REVIEW

File: docs/using-northstar/advanced.md [hunk 1]
```diff
@@ -17,17 +17,24 @@ You'll want to rename the `R2Northstar` folder to whatever you prefer it to be c
...
+
+Double clicking the `.bat` or right clicking on it and hitting `open` will launch the vanilla profile. Alternatively, you can follow the [Steamedit section](advanced.md#setting-up-different-launch-methods-with-steamedit) to have a "launch menu" when launching Titanfall 2 on Steam.
 
-Double clicking the `.bat` or right clicking on it and hitting `open` will launch the vanilla profile.
+### Setting up different launch methods with Steamedit
+
+You can set up profiles in an even more advanced way by setting up a way to use Steam to launch multiple different profiles from their newer launch menu that appears when you can launch a game in more ways than one. Follow the steps already listed [here](../installing-northstar/basic-setup.md#adding-alternate-launch-option-for-steam), then add `-profile=PROFILE FOLDER NAME HERE` to the arguments of the new option for NorthstarLauncher, the example below showing the profile `R2NorthstarCoreMods`.
```
Comment (GeckoEidechse) [inline docs/using-northstar/advanced.md]: Newline for better diff

```suggestion
You can set up profiles in an even more advanced way by setting up a way to use Steam to launch multiple different profiles from their newer launch menu that appears when you can launch a game in more ways than one.
Follow the steps already listed [here](../installing-northstar/basic-setup.md#adding-alternate-launch-option-for-steam), then add `-profile=PROFILE FOLDER NAME HERE` to the arguments of the new option for NorthstarLauncher, the example below showing the profile `R2NorthstarCoreMods`.
```

Also, the first sentence is kinda long and feels convoluted maybe take out filler words or just split up into individual sentences.

File: docs/using-northstar/advanced.md [hunk 2]
```diff
@@ -17,17 +17,24 @@ You'll want to rename the `R2Northstar` folder to whatever you prefer it to be c
...
-This tells the NorthstarLauncher to not restrict server commands, a feature which is needed to allow the server to tell the client which server to join (but is disabled on Northstar for security reasons), and tells it to use the `R2Vanilla` profile, which, as it doesn't normally exist, will launch Northstar with no core mods enabled, allowing you to easily play on Vanilla using most of Northstar's security fixes.
+This tells the NorthstarLauncher to not restrict server commands, a feature which is needed to allow the server to tell the client which server to join (but is disabled on Northstar for security reasons), and tells it to use the `R2Vanilla` profile.
+
+By default, this profile doesn't exist, meaning you won't load _any_ mods when using it.\
+However, you can still add mods to the profile while doing this. To do this, simply create a folder called `R2Vanilla`, then create a `mods` folder in there, and move the mods you want to use on vanilla into that mods folder. Note that not all mods will work this way, especially ones that change the game drastically, or that use the mod settings feature found in Northstar.
```
Comment (GeckoEidechse) [inline docs/using-northstar/advanced.md]: Newline in source for diff.

```suggestion
However, you can still add mods to the profile while doing this. To do this, simply create a folder called `R2Vanilla`, then create a `mods` folder in there, and move the mods you want to use on vanilla into that mods folder.
Note that not all mods will work this way, especially ones that change the game drastically, or that use the mod settings feature found in Northstar.
```

File: docs/using-northstar/advanced.md [hunk 3]
```diff
@@ -17,17 +17,24 @@ You'll want to rename the `R2Northstar` folder to whatever you prefer it to be c
...
+
+Double clicking the `.bat` or right clicking on it and hitting `open` will launch the vanilla profile. Alternatively, you can follow the [Steamedit section](advanced.md#setting-up-different-launch-methods-with-steamedit) to have a "launch menu" when launching Titanfall 2 on Steam.
 
-Double clicking the `.bat` or right clicking on it and hitting `open` will launch the vanilla profile.
+### Setting up different launch methods with Steamedit
+
+You can set up profiles in an even more advanced way by setting up a way to use Steam to launch multiple different profiles from their newer launch menu that appears when you can launch a game in more ways than one. Follow the steps already listed [here](../installing-northstar/basic-setup.md#adding-alternate-launch-option-for-steam), then add `-profile=PROFILE FOLDER NAME HERE` to the arguments of the new option for NorthstarLauncher, the example below showing the profile `R2NorthstarCoreMods`.
+
+![SteamEdit using Northstar Profiles](../images/steamedit-profiles.png)
```
Comment (GeckoEidechse) [inline docs/using-northstar/advanced.md]: That image seems to be missing from the PR ^^"

**Review by GeckoEidechse — COMMENTED** [review]
Overall I like the change. A few technicalities to address then we should be good to merge ^^

**Comment by itscynxx** [comment]
To be honest, Northstar without core mods is a little unplayable currently due to the rpc spam in console, without even getting into the entire modsetings being missing thing (and having no replacement other than vanilla+ now that mod settings is blacklisted; which is a very good thing, don't get me wrong, it just means that one small, specific sub section of Northstar becomes less compatible for the majority to become much much more compatible, which I'm all for)

I also don't find it particularly right for me to write a section shilling vanilla+ without some kind of green light
At the very least, I'd make sure to nuke it when Northstar gets better vanilla compatibility

At the very least 2 electric boogaloo, imo this shouldn't get merged until at least plugins v3 where vanilla has _playable_ framerates from rpc being fixed

**Comment by GeckoEidechse** [comment]
Aight, setting to draft then in the meantime.

**Comment by itscynxx** [comment]
With plugins v3, vanilla is (or at least should be, I haven't explicitly checked) playable again with no core mods (without making a dummy mod or using vanilla+ to stop the spam), so I think this is ready to be reviewed again

**Comment by itscynxx** [comment]
I'm going to close this due the changes being made in terms of vanilla compatibility

I'll create a separate pr with updated information when it's relevant, instead of having this merged and having another pr to update it again merged (hopefully) not too soon after

**Pull request closed without merging**